arch/arm/core/aarch32/cortex_m/Kconfig

```diff
@@ -86,18 +90,11 @@ config CPU_CORTEX_M_HAS_SYSTICK
 	help
 	  This option is enabled when the CPU implements the SysTick timer.
 
-config CPU_CORTEX_M_HAS_CACHE
-	bool
-	depends on CPU_CORTEX_M7 || CPU_CORTEX_M55
-	help
-	  This option signifies that the CPU implements Data and Instruction
-	  Cache
-
 config DCACHE_LINE_SIZE
-	default 32 if CPU_CORTEX_M_HAS_CACHE
+	default 32
```

Comment:
Just a remark, I think that in the **future** we should add that as a property in the device tree instead. For instance linux has the `d-cache-line-size` property for the CPU.

Reply:
We have already that: https://github.com/zephyrproject-rtos/zephyr/blob/0456ddc27d79abad516e2199c5ecc04340e73933/include/zephyr/cache.h#L141-L149

When neither `CONFIG_DCACHE_LINE_SIZE_DETECT` nor `CONFIG_DCACHE_LINE_SIZE` is selected, the cache line size is retrieved from the DT.